admin/commands/pmm_server_upgrade/run/run.go

```diff
@@ -18,35 +18,30 @@ package run
 import (
 	"context"
 	"fmt"
-	"net"
-	"os"
-	"time"
+	"path/filepath"
```

Comment:
Is it ok directory name with underscores like pmm_server_upgrade? most of them in project with dashes